Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -0,0 +1,666 @@
+        if comments:
+            request_data['comment'] = comments
+        request_data['filters'] = filters
+    elif custom_filter:
+        request_data = json.loads(custom_filter)
+    return request_data
+
+
+def test_module(client):
+    try:
+        client.list_alerts(url_suffix='/alerts/', request_data={})
+        if demisto.params().get('isFetch'):
+            client.list_incidents(filters={}, limit=1)
+    except Exception as e:
+        if 'No connection' in str(e):
+            return 'Connection Error: The URL you entered is probably incorrect, please try again.'
+        if 'Invalid token' in str(e):
+            return 'Authorization Error: make sure API Key is correctly set'
+        return str(e)
+    return 'ok'
+
+
+def alert_to_human_readable(alert):
+    readable_output = assign_params(alert_id=alert.get('_id'), title=alert.get('title'),
+                                    description=alert.get('description'), is_open=alert.get('is_open'),
+                                    status_value=[key for key, value in STATUS_OPTIONS.items()
+                                                  if alert.get('statusValue') == value],
+                                    severity_value=[key for key, value in SEVERITY_OPTIONS.items()
+                                                    if alert.get('severityValue') == value],
+                                    alert_date=datetime.fromtimestamp(alert.get('timestamp') / 1000.0).isoformat())
+    return readable_output
+
+
+def alerts_to_human_readable(alerts, alert_id):
+    if not alert_id:
+        alerts_readable_outputs = []
+        for alert in alerts:
+            alerts_readable_outputs.append(alert_to_human_readable(alert))
+    else:
+        alerts_readable_outputs = alert_to_human_readable(alerts)
+    headers = ['alert_id', 'alert_date', 'title', 'description', 'status_value', 'severity_value', 'is_open']
+    human_readable = tableToMarkdown('Microsoft CAS Alerts', alerts_readable_outputs, headers, removeNull=True)
+    return human_readable
+
+
+def arrange_alert_description(alert):
+    if alert.get('description') and '__siteIcon__' in alert.get('description'):
+        description = alert.get('description').replace('__siteIcon__', 'siteIcon')
+        alert['description'] = description
+
+
+def arrange_alerts_descriptions(alerts, alert_id):
+    if not alert_id:
+        for alert in alerts:
+            arrange_alert_description(alert)
+    else:
+        arrange_alert_description(alerts)
+    return alerts
+
+
+def set_alert_is_open(alert):
+    if alert.get('resolveTime'):
+        alert['is_open'] = False
+    else:
+        alert['is_open'] = True
+
+
+def set_alerts_is_open(alerts, alert_id):
+    if not alert_id:
+        for alert in alerts:
+            set_alert_is_open(alert)
+    else:
+        set_alert_is_open(alerts)
+    return alerts
+
+
+def list_alerts_command(client, args):
+    url_suffix = '/alerts/'
+    alert_id = args.get('alert_id')
```

Comment:
can't this be a list? if so you need to create a list from this(Same goes for the other places)